plugins/@grouparoo/postgres/src/lib/connect.ts

```diff
@@ -24,19 +36,17 @@ export const connect: ConnectPluginAppMethod = async ({ appOptions }) => {
   sslOptions.forEach((opt) => {
     if (formattedOptions[opt] !== null && formattedOptions[opt] !== undefined) {
       if (!formattedOptions["ssl"]) formattedOptions["ssl"] = {};
+      // @ts-ignore I think this is wrong, but don't want to mess with it
```

Comment:
Why is it screaming at you? 

I would guess something about the `ssl` obj within the `formattedOptions` type above not having `cert`, `key` and `ca`? Would adding those to the type and doing `const sslOptions = ["ssl_cert", "ssl_key", "ssl_ca"] as const` work? not sure if it's smart enough to work out the `.replace()`